web/index.php

```diff
@@ -40,9 +39,41 @@
   'monolog.logfile' => 'php://stderr',
 ));
 
+$config = [
+  'secret' => getenv('SECRET'),
+  'api_token' => getenv('GITHUB_API_TOKEN'),
+  'jira_url' => getenv('JIRA_URL'),
+  'jira_username' => getenv('JIRA_USERNAME'),
+  'jira_password' => getenv('JIRA_PASSWORD'),
+  'issue_prefix' => getenv('JIRA_ISSUE_PREFIX'),
+  'transition' => [
+    'opened' => [
+      'id' => filter_var(
+        getenv('JIRA_TRANSITION_OPENED'),
+        FILTER_VALIDATE_INT
+      ),
+      'fields' => json_decode(getenv('JIRA_TRANSITION_OPENED_FIELDS'), true)
+    ],
+    'closed' => [
+      'id' => filter_var(
+        getenv('JIRA_TRANSITION_CLOSED'),
+        FILTER_VALIDATE_INT
+      ),
+      'fields' => json_decode(getenv('JIRA_TRANSITION_CLOSED_FIELDS'), true)
+    ],
+    'merged' => [
+      'id' => filter_var(
+        getenv('JIRA_TRANSITION_MERGED'),
+        FILTER_VALIDATE_INT
+      ),
+      'fields' => json_decode(getenv('JIRA_TRANSITION_MERGED_FIELDS'), true)
+    ]
+  ]
+];
+
 // Handle the post request
-$app->post('/', function(Request $request) use ($app) {
-  $hook = new Webhook($app, $request);
+$app->post('/', function(Request $request) use ($app, $config) {
```

Comment:
```php
$app->post(
  '/',
  function . . .
);

Reply:
This doesnt exceed line length limitations.

Reply:
no; referring to having two args on one line when the call covers multiple lines.  when args span multiple lines, each arg must be on its own line.

Reply:
I dont think that rule should apply for closures. it honestly looks terrible when put on multiple lines line that and is not any more readable.

@robertology ?